policy/modules/system/init.te

```diff
@@ -468,6 +468,10 @@ ifdef(`init_systemd',`
 	selinux_compute_user_contexts(init_t)
 	selinux_use_status_page(init_t)
 
+	# systemd loads the policy itself instead of using load_policy
+	selinux_load_policy(init_t)
+	seutil_read_bin_policy(init_t)
+
```

Comment:
Does not make sense though... when systemd loads policy, selinux is not even enabled yet, so it wouldnt need that permission. Why would systemd load_policy if selinux is already enabled? Seems questionable...

Reply:
I think this might be related to using the dracut SELinux module which tries to load the policy much earlier in the boot process. Still investigating though.